services/movie.py

```diff
@@ -0,0 +1,41 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        ).distinct()
+    if actors_ids:
+        return Movie.objects.filter(
+            actors__id__in=actors_ids
+        ).distinct()
+    if genres_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids
+        ).distinct()
```

Comment:
You can simplify this function
```suggestion
    query = Movie.objects.all()
    if actors_ids:
        query = query.filter(actors__id__in=actors_ids)
    if genres_ids:
        query = query.filter(genres__id__in=genres_ids)
    return query.distinct()
```